src/amo/components/GuidesAddonCard/index.js

```diff
@@ -39,6 +40,13 @@ export class GuidesAddonCardBase extends React.Component<InternalProps> {
   render() {
     const { addon, i18n, staffPick } = this.props;
 
+    const fallback =
+      addon !== null ? (
+        <Card>
```

Comment:
yeh that's better. thanks for the suggestion

Reply:
I only am doing this with the title since that is really the only thing that we are getting from fetching the addon (except for the image which isn't text).

Reply:
Great, I really like it!